src/cli/cluster/worker.js

```diff
@@ -114,8 +136,13 @@ module.exports = class Worker extends EventEmitter {
     }
 
     this.fork = cluster.fork(this.env);
-    this.fork.on('message', this.onMessage);
+    this.fork.on('message', this.parseIncomingMessage);
```

Comment:
I suspect this needs to be explicitly bound to `this` otherwise it'll be run in the context of the message event itself.